core/src/test/java/gnu/trove/MapTest.java

```diff
@@ -354,6 +355,31 @@ public boolean execute(Integer a, String b) {
         assertEquals((int) (map.capacity() * map._loadFactor), map._maxSize);
     }
 
+    public static void testTHashMapViewsEquality() {
+      THashMap<String, Integer> oneMap = new THashMap<String, Integer>();
+      THashMap<String, Integer> twoMap = new THashMap<String, Integer>();
```

Comment:
It would be good to compare with non-THashMap implementation (e.g. usual HashMap) in test as well.